devtools/credits.pl

```diff
@@ -770,6 +770,7 @@ sub add_paragraph {
 				add_person("Max Horn", "Fingolfin", "(retired)");
 				add_person("Oystein Eftevaag", "vinterstum", "(retired)");
 				add_person("Thierry Crozat", "criezy", "");
+				add_person("dwa", "dwa", "Tiger/Leopard PPC");
```

Comment:
```suggestion
				add_person("", "dwa", "Tiger/Leopard PPC");
```

Just leave the first string (real name) empty (and rerun `make credits` to update the other files).

This will not change anything for the generated `AUTHORS`, `credits.rst`, and `credits.h` files, but will for the `credits.yaml` file used on the website.

Reply:
Fixed, thanks!